### PR TITLE
Mode support

### PR DESCRIPTION
**Pull request by erabinov**
#### Reference Issue
Multiple accidentals in tonics, expanded support for modes in note spelling #1724

#### What does this implement/fix? Explain your changes.

I made some editing tweaks to the documentation to reflect the changes made in PR #1756 to the code.

### CODE REVIEW

File: docs/examples/plot_display.py [hunk 1]
```diff
@@ -178,7 +178,7 @@
 fig.colorbar(img, ax=ax)
 
 # %%
-# If you also happen to know the key of the piece being analyzed, you can
+# If you also happen to know the key or mode of the piece being analyzed, you can
 # pass this to specshow and it will spell the notes properly:
 fig, ax = plt.subplots()
 img = librosa.display.specshow(chroma, y_axis='chroma', x_axis='time',
```
Comment (bmcfee) [inline docs/examples/plot_display.py:184]: While we're on this, could we expand `F:dor` to `F:dorian` (just to be a bit more reader-friendly)?

Comment (erabinov) [reply docs/examples/plot_display.py:184]: Sounds good!

**Comment by bmcfee** [comment]
LGTM - anything else to do here, or can this merge when CI finishes?

**Comment by erabinov** [comment]
Should be good to go. Thanks!

**Review by bmcfee — APPROVED** [review]

**Comment by bmcfee** [comment]
Thanks again!

**Pull request merged**